crates/ra_hir/src/code_model.rs

```diff
@@ -769,6 +818,7 @@ impl TypeParam {
     }
 }
 
+// FIXME: rename to `ImplBlock`
```

Comment:
It already is?

Reply:
s/to/from, it should be Impl :D

Reply:
I'll adjust this in my other PR, seems not to be worth of a separate PR.